README.md

```diff
@@ -673,10 +673,13 @@ Default configuration search paths (see all with `grype config locations`):
 Use `grype config` to print a sample config file to stdout.
 Use `grype config --load` to print the current config after loading all values to stdout.
 
-You can specify files directly using the `--config` / `-c` flags to provide your own configuration files/paths:
+You can specify files directly using the environment variable `GRYPE_CONFIG` or the `--config` / `-c` flags to provide your own configuration files/paths:
```

Comment:
I don't think we need to include an example of this, since someone specifying this on the command line probably would just use `-c` and that would be the preferred way rather than using an environment variable that could be overridden. I think it would make sense to revert the dependency here, but add a blurb secondary to the `-c`, so something like: 

> You can specify files directly using the `--config` / `-c` flags (or environment variable `GRYPE_CONFIG`) to provide your own configuration files/paths:

... would this have been sufficient for you to discover this detail?

Reply:
Completely agree! Fixed